lib/ui/page/home/page/chat/info/view.dart

```diff
@@ -579,4 +610,86 @@ class ChatInfoView extends StatelessWidget {
       await c.reportChat();
     }
   }
+
+  /// Returns the action buttons for monolog
+  Widget _actionsMonolog(ChatInfoController c, BuildContext context) {
+    return Block(
+      children: [
+        Obx(() {
+          final bool favorite = c.chat?.chat.value.favoritePosition != null;
+          return Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              ActionButton(
+                key: Key(
+                  favorite ? 'UnfavoriteChatButton' : 'FavoriteChatButton',
+                ),
+                text: favorite
+                    ? 'btn_delete_from_favorites'.l10n
+                    : 'btn_add_to_favorites'.l10n,
+                trailing: SvgIcon(
+                  favorite ? SvgIcons.favoriteSmall : SvgIcons.unfavoriteSmall,
+                ),
+                onPressed: favorite ? c.unfavoriteChat : c.favoriteChat,
+              ),
+              ActionButton(
+                key: const Key('ClearHistoryButton'),
+                text: 'btn_clear_chat'.l10n,
+                trailing: const SvgIcon(SvgIcons.cleanHistory),
+                onPressed: () => _clearChat(c, context),
+              ),
+              ActionButton(
+                key: const Key('HideChatButton'),
+                text: 'btn_delete_chat'.l10n,
+                trailing: const SvgIcon(SvgIcons.delete),
+                onPressed: () => _hideChat(c, context),
+              ),
```

Comment:
Doesn't this icon seem out of place for you?

<img width="483" alt="Screenshot 2025-05-28 at 10 26 14" src="https://github.com/user-attachments/assets/d084ce11-cc4d-43dc-82d2-63fa57139e59" />